support/defaults.go

```diff
@@ -5,7 +5,9 @@ package support
 // ***********************
 
 const (
-	RayVersion   = "2.35.0"
-	RayImage     = "quay.io/modh/ray:2.35.0-py39-cu121"
-	RayROCmImage = "quay.io/modh/ray:2.35.0-py39-rocm61"
+	RayVersion        = "2.35.0"
+	RayImage          = "quay.io/modh/ray:2.35.0-py39-cu121"
+	RayROCmImage      = "quay.io/modh/ray:2.35.0-py39-rocm61"
+	RayTorchCudaImage = "quay.io/rhoai/ray:2.35.0-py39-cu121-torch24-sh20"
```

Comment:
Out-of-curiosity, what is the Dockerfile that's been used to build that image?

Reply:
btw, what is the difference between `2.35.0-py39-cu121-torch24-fa26` and `2.35.0-py39-cu121-torch24-sh20`?

Reply:
[Docker file](https://github.com/opendatahub-io/distributed-workloads/tree/main/images/runtime/examples/ray-torch-cuda) for ray-torcj-cuda image
[Docker file](https://github.com/opendatahub-io/distributed-workloads/tree/main/images/runtime/examples/ray-torch-rocm) for ray-torch-rocm image

Reply:
`2.35.0-py39-cu121-torch24-sh20` includes recent changes where 
Base image is changed `quay.io/rhoai/ray:2.23.0-py39-cu121` to  `quay.io/modh/ray:2.35.0-py39-cu121` in [dockerfile](2.35.0-py39-cu121-torch24-sh20) 
@sutaakar

Reply:
@ChughShilpa I see, then I think we can stick to `2.35.0-py39-cu121-torch24-fa26` to be consistent. If you've found out that this tag does not reflect the latest state of the Dockerfile, then it can be rebuilt and re-pushed from the latest version. WDYT?

Reply:
I like the approach of sticking with consistent tag. I will rebuilt the image with latest dockerfile and re-pushed it

Reply:
Rebuilt the image and added the same tag `2.35.0-py39-cu121-torch24-fa26`